Services/Facades/PlacedBetFacade.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AutoMapper;
+using Domain;
+using Domain.Dto.BetableEntity;
+using Domain.Dto.BetQuote;
+using Domain.Dto.BetRequest;
+using Domain.Dto.Bets;
+using Domain.Dto.FullBetView;
+using Domain.Dto.PlacedBet;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Services.Interfaces;
+
+namespace Services.Facades
+{
+    public class PlacedBetFacade : IPlacedBetFacade
+    {
+        private readonly IPlacedBetsService _placesService;
+        private readonly IBetQuoteService _betQuoteService;
+        private readonly IBetsService _betService;
+        private readonly IBetableEntityService _betableEntityService;
+        private readonly IMapper _mapper;
+
+        public PlacedBetFacade(IPlacedBetsService placedBetsService,
+                               IBetQuoteService betQuoteService,
+                               IBetsService betsService,
+                               IBetableEntityService betableService,
+                               IMapper mapper) 
+        {
+            _placesService = placedBetsService;
+            _betQuoteService = betQuoteService;
+            _betService = betsService;
+            _betableEntityService = betableService;
+            _mapper = mapper;
+        }
+
+        private bool ValidateType(BetOptions type)
+        {
+            return Enum.IsDefined(typeof(BetOptions),type);
+        }
```

Comment:
this check doesn't make sense ... this will be validating by asp.net when parsing the requests body